tests/staticfiles_tests/test_management.py

```diff
@@ -110,6 +111,13 @@ def test_all_files_more_verbose(self):
             searched_locations
         )
 
+    @override_settings(STATICFILES_DIRS=('test'))
+    def test_check(self):
+        err = StringIO()
+        with self.assertRaises(SystemCheckError):
+            call_command('collectstatic', skip_checks=False, stderr=err)
+        self.assertIn(err.getvalue(), 'The STATICFILES_DIRS setting is not a tuple or list')
```

Comment:
This test doesn't work properly because we check if `'' in 'The STATICFILES_DIRS...'` which is always true.